src/components/contributions-table.jsx

```diff
@@ -131,6 +131,16 @@ class ContributionsTable extends React.Component {
       });
     };
 
+    const isActive = (colName) => {
+      const currentSort = this.state.sort;
+      if (this.state.sort.column !== colName) {
+        return '';
+      } else if (currentSort.order === 1) {
+        return ' down-arrow';
```

Comment:
For state-related classes, let's use `is-` prefix. so like, `is-ascending`  or `is-descending`

Reply:
Sure thing!